Firestore/core/src/firebase/firestore/util/status.h

```diff
@@ -57,6 +58,8 @@ class ABSL_MUST_USE_RESULT Status {
   /// Creates a status object from the given errno error code and message.
   static Status FromErrno(int errno_code, absl::string_view message);
 
+  static Status FromGrpcStatus(const grpc::Status& from);
```

Comment:
This differs from the errno and last error cases in that it requires some other library to do. To date, no library besides remote has needed any dependency on grpc. I think this would be better if it paralleled the way we did it for leveldb:

https://github.com/firebase/firebase-ios-sdk/blob/master/Firestore/core/src/firebase/firestore/local/leveldb_util.h#L42

Reply:
Done, thanks!